### PR TITLE
[#158375013] deploy redis to vnet

### PR DESCRIPTION
**Pull request by gunzip**
creates a new virtual network for redis cache

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * ⚙️ [#158375013](https://www.pivotaltracker.com/story/show/158375013): La risorsa Redis deve risiedere sulla vnet del cluster kubernetes

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: infrastructure/azure.tf [hunk 1]
```diff
@@ -1032,10 +1032,10 @@ resource "azurerm_redis_cache" "azurerm_redis_cache" {
...
   # see https://docs.microsoft.com/en-us/azure/redis-cache/cache-faq#what-redis-cache-offering-and-size-should-i-use
   redis_configuration {
-    # Value in megabytes reserved for non-cache usage e.g. failover
-    maxmemory_reserved = 64
-    maxmemory_delta    = 64
-    maxmemory_policy   = "allkeys-lru"
+    # see https://www.terraform.io/docs/providers/azurerm/r/redis_cache.html#default-redis-configuration-values
+    maxmemory_reserved = 200
+    maxmemory_delta    = 200
+    maxmemory_policy   = "volatile-lru"
```
Comment (cloudify) [inline infrastructure/azure.tf]: describe what this setting does

**Comment by cloudify** [comment]
@gunzip what's the status of this PR?

**Comment by gunzip** [comment]
it lacks the DNS configuration, atm it uses a static ip address

**Comment by cloudify** [comment]
ok I think we can live with that for now

**Pull request merged**